### PR TITLE
Add FAQ section

### PR DESCRIPTION
**Pull request by helfi92**
Added question asked from [https://hacks.mozilla.org/2017/02/using-neutrino-for-modern-javascript-development/#comment-20663](https://hacks.mozilla.org/2017/02/using-neutrino-for-modern-javascript-development/#comment-20663)

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -23,3 +23,4 @@
 * [Contributing](/contributing/README.md)
   * [Development Process](/contributing/development.md)
   * [Code of Conduct](/contributing/code-of-conduct.md)
+* [FAQ](/FAQ.md)
```
Comment (eliperelman) [inline docs/SUMMARY.md]: Let's move FAQs higher up in the summary, maybe before "Presets".

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# FAQ
+
+### What is the added value versus all the boilerplate projects out there like [create-react-app](https://github.com/facebookincubator/create-react-app)?
+
+The proliferation of boilerplate and metapackages is one thing we are trying to reduce. These types of projects
```
Comment (eliperelman) [inline docs/FAQ.md]: :heart_eyes:

**Comment by jnachtigall** [comment]
@helfi92 @eliperelman I was the one opening #50 after reading the question at hacks.mozilla.org. TBH  I found the answer at https://github.com/mozilla-neutrino/neutrino-dev/issues/50#issuecomment-282316271 much more comprehensible than the one given here. With it it made "click" in my head, with the answer at hacks I was just shrugging my shoulders.

**Comment by eliperelman** [comment]
I'll move my blurb from the presets page to the FAQs page.

**Pull request merged**